packages/wonder-blocks-popover/src/components/__tests__/initial-focus.test.js

```diff
@@ -1,6 +1,8 @@
 // @flow
 import * as React from "react";
-import {mount} from "enzyme";
+import {render, screen} from "@testing-library/react";
```

Comment:
I took the opportunity to migrate these tests to RTL while figuring out the NodeList spread issue.